docs/adguard-browser-extension/solving-problems/update-rules.md

```diff
@@ -0,0 +1,94 @@
+---
+title: How to update declarative rules
```

Comment:
This is about debugging rules, not updating.

`Debugging filtering rules in an MV3 extension`